scripts/alpine-setup.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env sh
 
 # Set strict error checking
-set -emou pipefail
+set -eou pipefail
```

Comment:
Is the `-m` option causing any trouble?

Reply:
job control disabled in docker image

Reply:
Ok, we're not using it anyway, so I guess that's fine.